packages/twenty-front/src/modules/favorites/components/CurrentWorkspaceMemberFavorites.tsx

```diff
@@ -1,127 +1,229 @@
+import { Dropdown } from '@/ui/layout/dropdown/components/Dropdown';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { useDropdown } from '@/ui/layout/dropdown/hooks/useDropdown';
+import { ConfirmationModal } from '@/ui/layout/modal/components/ConfirmationModal';
+import { NavigationDrawerInput } from '@/ui/navigation/navigation-drawer/components/NavigationDrawerInput';
 import { NavigationDrawerItem } from '@/ui/navigation/navigation-drawer/components/NavigationDrawerItem';
-import { NavigationDrawerSection } from '@/ui/navigation/navigation-drawer/components/NavigationDrawerSection';
-import { NavigationDrawerSectionTitle } from '@/ui/navigation/navigation-drawer/components/NavigationDrawerSectionTitle';
-import { useNavigationSection } from '@/ui/navigation/navigation-drawer/hooks/useNavigationSection';
-
-import { currentWorkspaceMemberState } from '@/auth/states/currentWorkspaceMemberState';
-import { NavigationDrawerAnimatedCollapseWrapper } from '@/ui/navigation/navigation-drawer/components/NavigationDrawerAnimatedCollapseWrapper';
 import { NavigationDrawerItemsCollapsedContainer } from '@/ui/navigation/navigation-drawer/components/NavigationDrawerItemsCollapsedContainer';
-import { ScrollWrapper } from '@/ui/utilities/scroll/components/ScrollWrapper';
-import { useFavorites } from '../hooks/useFavorites';
-
-const StyledContainer = styled(NavigationDrawerSection)`
-  width: 100%;
-`;
-
-const StyledAvatar = styled(Avatar)`
-  :hover {
-    cursor: grab;
-  }
-`;
-
-const StyledNavigationDrawerItem = styled(NavigationDrawerItem)`
-  :active {
-    cursor: grabbing;
-
-    .fav-avatar:hover {
-      cursor: grabbing;
-    }
-  }
-`;
-
-export const CurrentWorkspaceMemberFavorites = () => {
-  const currentWorkspaceMember = useRecoilValue(currentWorkspaceMemberState);
-
-  const { favorites, handleReorderFavorite } = useFavorites();
-  const loading = useIsPrefetchLoading();
-  const { toggleNavigationSection, isNavigationSectionOpenState } =
-    useNavigationSection('Favorites');
-  const isNavigationSectionOpen = useRecoilValue(isNavigationSectionOpenState);
-
-  if (loading && isDefined(currentWorkspaceMember)) {
-    return <FavoritesSkeletonLoader />;
-  }
+import { NavigationDrawerSubItem } from '@/ui/navigation/navigation-drawer/components/NavigationDrawerSubItem';
+import { getNavigationSubItemState } from '@/ui/navigation/navigation-drawer/utils/getNavigationSubItemState';
+import { useTheme } from '@emotion/react';
+import { useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import {
+  IconDotsVertical,
+  IconFolder,
+  IconHeartOff,
+  IconPencil,
+  IconTrash,
+  MenuItem,
+} from 'twenty-ui';
+
+type CurrentWorkspaceMemberFavoritesProps = {
+  folder: {
+    folderId: string;
+    folderName: string;
+    favorites: ProcessedFavorite[];
+  };
+  isGroup: boolean;
```

Comment:
can isGroup be infered from folder.favorites length?

Reply:
isOpen, onToggle do not need to be passed by the parent, they can be computed / used in the packages/twenty-front/src/modules/favorites/components/CurrentWorkspaceMemberFavorites.tsx component directly